### PR TITLE
feat(beta): add badge support for beta properties and examples

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR adds support for badges to represent beta properties and examples. Should help facilitate this design https://github.com/patternfly/patternfly-org/issues/1646#issuecomment-582939063

### CODE REVIEW

File: packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const AutoLinkHeader = ({
   size,
   headingLevel,
   children,
+  metaText,
```
Comment (redallen) [inline packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js]: This prop is essentially `children`, there's no need for it.

File: packages/patternfly-4/gatsby-theme-patternfly-org/components/example/example.js [hunk 2]
```diff
@@ -97,7 +98,7 @@ export const Example = props => {
 
   return (
     <div className="ws-example">
-      <AutoLinkHeader size="h4" headingLevel="h3" className="ws-example-heading">
+      <AutoLinkHeader metaText={isBeta ? <Badge className="ws-beta-badge pf-u-ml-xs">Beta</Badge> : null} size="h4" headingLevel="h3" className="ws-example-heading">
         {exampleName}
```
Comment (redallen) [inline packages/patternfly-4/gatsby-theme-patternfly-org/components/example/example.js]: Instead you can change `{exampleName}` to:
```
{exampleName}
{isBeta && <Badge className="ws-beta-badge pf-u-ml-xs">Beta</Badge>}
```

File: packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js (hunk 1 above)
Comment (seanforyou23) [reply packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js]: Using `children` produces the following fragment when selecting the anchor

Screen Shot 2020-02-10 at 1 13 30 PM

Comment (redallen) [reply packages/patternfly-4/gatsby-theme-patternfly-org/components/autoLinkHeader/autoLinkHeader.js]: Ah, good point! I forgot we serialize children for the anchor link.

**Comment by seanforyou23** [comment]
Forgot to link this PR to a corresponding issue in react - https://github.com/patternfly/patternfly-react/issues/3624

**Pull request merged**